apps/remote_control/test/lexical/remote_control/code_mod/format_test.exs

```diff
@@ -125,6 +125,28 @@ defmodule Lexical.RemoteControl.CodeMod.FormatTest do
     end
   end
 
+  describe "format/2 syntax correction" do
+    test "adds missing space after keyword", %{project: project} do
+      assert {:ok, result} = ~q[
+        %{foo:bar}
+      ] |> modify(project: project)
+
+      assert result == ~q[
+        %{foo: bar}
+      ]t
+    end
+
+    test "adds missing space after multiple keywords", %{project: project} do
+      assert {:ok, result} = ~q[
+        %{foo:bar, baz:buzz}
```

Comment:
I'm all for this change, but will its original format function work simultaneously?

Reply:
I'm not sure I understand the question. What do you mean by "work simultaneously?"

The way this works now is by trying to format, and in the event of an error, checking if there's a "known fix" available for it. It then calls the same format function again recursively, so a second error may lead to a second fix, and so on. If the formatter returns successfully the first time, no fixes are attempted.

Reply:
No worries! The recursion is a bit non-obvious, since it's re-wrapping the formatting function with itself before formatting the corrected code:

```elixir
wrap_with_syntax_corrections(formatter_fn).(corrected)
```

If there's a better/cleaner way to structure this, I'm all for it!